sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -298,4 +298,32 @@ class DataFrameFunctionsSuite extends QueryTest {
       Seq(Row(2), Row(0), Row(3))
     )
   }
+
+  test("array contains function") {
+    val df = Seq(
+      (Seq[Int](1, 2), "x"),
+      (Seq[Int](), "y"),
+      (null, "z")
+    ).toDF("a", "b")
+    checkAnswer(
+      df.select(array_contains("a", 1)),
+      Seq(Row(true), Row(false), Row(false))
+    )
+    checkAnswer(
+      df.selectExpr("array_contains(a, 1)"),
+      Seq(Row(true), Row(false), Row(false))
+    )
+    //checkAnswer(
```

Comment:
So given that I should use `eval` since we are overriding the default behavior of `null` arguments (which is being overridden to return `false`), and that as the tests are currently written they fail (with the above stack trace), how would it be best to test this?